### DISCUSSION TITLE
Awesome work, could guide me to stand up please~

### DISCUSSION THREAD

**Discussion by Elkaroui** [uicrooks/shopify-theme-lab#58] [Q&A] [answered]
Hi, first I want to thank you and all team for this amazing work, I am new to Vue and my last week was hell for me to integrated Vuejs + Vuetify with Shopify, I had many faults and bugs, and finally, I find your project that works just perfectly {{Love it}}.

However, could please teach me to know how to add integrate some Vue plugging for example how we can add this code correctly:

```javascript
<template>
  <div class="demo">
    <image-shadow :width="280" :src="this.src1"></image-shadow>
    <image-shadow :width="280" :src="this.src2"></image-shadow>
    <image-shadow :width="280" :shadow-radius="20" :src="this.src3"></image-shadow>
  </div>
</template>

<script>
export default {
  data() {
    return {
      src1: 'https://github.com/image-component/gallery/blob/main/girl/5.jpg?raw=true',
      src2: 'https://github.com/image-component/gallery/blob/main/girl/1.jpg?raw=true',
      src3: 'https://github.com/image-component/gallery/blob/main/girl/3.jpg?raw=true'
    }
  }
}
</script>
```

also for what I understand, we need to add delimiters
```javascript
delimiters: ['${', '}']
```
Could you please guide me to know how add this one in team.liquid for example:

```javascript
<div class="flower">
  <h1>${name}</h1>
</div>
<script>
  new Vue({
    el: '.flower',
    delimiters: ['${', '}']
    data() {
      return {
        name: 'You are awesome!'
      }
    },
  
  })
</script>
```

**Comment by sergejcodes** [comment, marked as answer]
Hi there 👋,

you don't need to change the delimiters if you use Vue single file components or use the liquid [raw tag](https://shopify.github.io/liquid/tags/raw). 

As for your two code examples they are quite similar, here is one way to integrate them:

1. Create a file. For example `src/vue/components/render/simple-text.vue` it will be auto-loaded into the global Vue instance
2. Add the following content:
```vue
<template>
  <div>
    <h1>{{ name }}</h1>
  </div>
</template>

<script>
export default {
  name: 'SimpleText', // if you provide a name, you can reference the component directly with kebab-case
  data () {
    return {
      name: 'You are awesome!'
    }
  }
}
</script>
```
3. Add `<simple-text></simple-text>` component to any `.liquid` file and it should render the Vue file

**Reply by Elkaroui** [reply]
Awesome, Thank you so much for the support ❤
